### ISSUE TITLE
Is it impossible to add InfluxDB 2.x Geth Metircs?

### ISSUE THREAD

**Issue by doogunwo** [ethereum/go-ethereum#29953]
When conducting research with Geth version 1.10.26, it was possible to add metrics.

![image](https://github.com/ethereum/go-ethereum/assets/87505243/e20f1887-c1d6-48f6-9a68-8c6d79a37895)

This code adds the currently commented part to numbers 154 to 156 of ethdb/leveldb/leveldb.go, and places db.metrics_name.Inc(1) where you want. When InfluxDB 1.x, the metrics are I was able to confirm that it was added to InfluxDB and data was being collected. However, adding the metric I want does not work in the latest version of Geth (currently 1.14.x). Is there another way? Any help would be greatly appreciated.

**Comment by jwasinger** [comment]
Please link the full code to reproduce this.  You've linked a snippet which introduces two new metrics, but doesn't actually show how you are populating them in the code.

**Comment by jwasinger** [comment]
Actually, closing this as it isn't an issue with Geth, but your custom changes.

**Comment by doogunwo** [comment]
![image](https://github.com/ethereum/go-ethereum/assets/87505243/d335027c-3d7f-4a65-ace9-f09d806bd450)
First of all I added to the Database struct: 2 new metrics

and I added 
ldb.leveldbWritebytes =  metrics.NewRegisteredCounter(namespace+"geth/leveldb/writebytes", nil)
ldb.leveldbGetCounter =  metrics.NewRegisteredCounter(namespace+"geth/leveldb/gets", nil)
Added these 2 to NewCustom.

Up to this point, matrix registration has been performed in Geth version 1.10.26. Why can't I register in the current Geth version 1.14.5?
